e2e/nomostest/testpredicates/predicates.go

```diff
@@ -289,14 +291,16 @@ func DeploymentContainerResourcesAllEqual(scheme *runtime.Scheme, expectedByName
 		}
 		// Validate the container resources exactly match expectations
 		var foundContainers []string
+		var errs []error
 		for _, container := range d.Spec.Template.Spec.Containers {
 			foundContainers = append(foundContainers, container.Name)
 			expectedSpec, ok := expectedByName[container.Name]
 			if !ok {
 				continue // error later when the list doesn't match
 			}
 			if err := validateContainerResources(&container, expectedSpec); err != nil {
-				return err
+				errs = append(errs, err)
+				continue
```

Comment:
nit: superfluous `continue`

Reply:
but it looks so pretty there and consistent with the other continue!